test/spec/xml/read.js

```diff
@@ -81,29 +81,39 @@ describe('read', function() {
     });
 
 
-    describe('zeebe:Subscription', function() {
```

Comment:
I don't quite get it. Is the subscription gone? Or is this an oversight that the test case is removed?

Reply:
It was a duplicate: https://github.com/camunda-cloud/zeebe-bpmn-moddle/pull/31/commits/441d76ad60c7a43a618ea1edeadeefe690c759f7

And GitHub uses it as replacement 😄

Reply:
🤦 on me

Reply:
:+1: